coral-trino/src/main/java/com/linkedin/coral/trino/rel2trino/transformers/SqlSelectListTransformer.java

```diff
@@ -0,0 +1,55 @@
+/**
+ * Copyright 2023 LinkedIn Corporation. All rights reserved.
+ * Licensed under the BSD-2 Clause license.
+ * See LICENSE in the project root for license information.
+ */
+package com.linkedin.coral.trino.rel2trino.transformers;
+
+import java.util.ArrayList;
+import java.util.List;
+
+import org.apache.calcite.sql.SqlCall;
+import org.apache.calcite.sql.SqlIdentifier;
+import org.apache.calcite.sql.SqlKind;
+import org.apache.calcite.sql.SqlNode;
+import org.apache.calcite.sql.SqlNodeList;
+import org.apache.calcite.sql.SqlSelect;
+import org.apache.calcite.sql.fun.SqlStdOperatorTable;
+import org.apache.calcite.sql.validate.SqlValidatorUtil;
+
+import com.linkedin.coral.common.transformers.SqlCallTransformer;
+
+import static org.apache.calcite.rel.rel2sql.SqlImplementor.*;
+
+
+/**
+ * This transformer operates on SqlSelect type sqlCalls. It appends an alias to all the projected fields with fully qualified names.
```

Comment:
Could you add an example and a justification for the transformation in the comment?

Reply:
I have added an example in the java doc. I have also updated the PR description to explain the motivation. Tldr: This is a migration from RelNode -> SqlNode